src/common/node/childProcess.ts

```diff
@@ -38,10 +38,16 @@ interface ISpawnOptions {
 
 export class ChildProcess {
     public static ERROR_TIMEOUT_MILLISECONDS = 300;
+    private childProcess: typeof child_process;
+
+    constructor({childProcess = child_process} = {}) {
```

Comment:
Now it looks like:
constructor({childProcess = child_process /* = require("child_process") */}